src/deploy/functions/release/index.ts

```diff
@@ -93,6 +93,12 @@ export async function release(
   const allErrors = summary.results.filter((r) => r.error).map((r) => r.error) as Error[];
   if (allErrors.length) {
     const opts = allErrors.length === 1 ? { original: allErrors[0] } : { children: allErrors };
+    console.log("Functions deploy failed.");
+    logger.info("Functions deploy failed.");
+    for (const error of allErrors) {
+      console.log(JSON.stringify(error, null, 2));
+      logger.info(JSON.stringify(error, null, 2));
+    }
```

Comment:
These probably need to be simplified in some way? Maybe `logger.debug` the errors if you like, but my hunch is that the console.log and logger.info should be removed

Reply:
Done.